### ISSUE TITLE
Remove a streamApp from the pipeline should not also remove its jar

### ISSUE THREAD

**Issue by konekoya** [oharastream/ohara#1129]
### _Please be aware that this issue was discovered in #1125 which is an ongoing issue and hasn't been merged yet_

When removing a StreamApp from the pipeline. The API also removes its jar from the StreamApp list page

Reproduce steps
1. Create necessary services, this includes zookeeper, broker, worker and a manager
2. From our UI, create a new pipeline
3. Inside the pipeline, add a stream app from the stream app list (You'll need to upload a stream app jar first)
4. Click on the newly created stream app and use the delete button on its upper right hand corner to remove it.

It will not only remove the graph from the pipeline but also remove its very jar from the list page as well.

**Comment by saivirtue** [comment]
related to #1125

**Comment by saivirtue** [comment]
It is because that the jar config and the streamApp config is saved to the same object in configurator ; The delete operation will remove jar also. This should be fixed after #1135

**Comment by konekoya** [comment]
Hmm, did a quick look. I think some of the props from the API response body has changed. And the UI has broken a little. I'll file another issue to tweak the UI...

**Comment by konekoya** [comment]
Issue opened #1167

**Comment by konekoya** [comment]
Closing this and in favor of #1167
